task/06-conditions-n-loops-tasks.js

```diff
@@ -192,7 +208,12 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+    let start = Math.min(a, b);
+    let end = Math.max(a, b);
+    if (isStartIncluded === false && isEndIncluded === false) return `(${start}, ${end})`;
```

Comment:
`if (!isStartIncluded && !isEndIncluded) return `(${start}, ${end})`;`
Please , rework the lines below this way

Please, see https://javascript.info/logical-operators
Please, read about https://developer.mozilla.org/en-US/docs/Glossary/Truthy